src/abstractdataframe/abstractdataframe.jl

```diff
@@ -404,33 +404,35 @@ function StatsBase.describe(io, df::AbstractDataFrame)
         println(io, )
     end
 end
-StatsBase.describe(dv::AbstractArray) = describe(STDOUT, dv)
-function StatsBase.describe{T<:Number}(io, dv::AbstractArray{T})
-    if all(isna(dv))
-        println(io, " * All NA * ")
-        return
-    end
-    filtered = float(dropna(dv))
-    qs = quantile(filtered, [0, .25, .5, .75, 1])
-    statNames = ["Min", "1st Qu.", "Median", "Mean", "3rd Qu.", "Max"]
-    statVals = [qs[1:3]; mean(filtered); qs[4:5]]
-    for i = 1:6
-        println(io, string(rpad(statNames[i], 8, " "), " ", string(statVals[i])))
-    end
-    nas = sum(isna(dv))
-    println(io, "NAs      $nas")
-    println(io, "NA%      $(round(nas*100/length(dv), 2))%")
-    return
-end
-function StatsBase.describe{T}(io, dv::AbstractArray{T})
-    ispooled = isa(dv, PooledDataVector) ? "Pooled " : ""
-    # if nothing else, just give the length and element type and NA count
-    println(io, "Length  $(length(dv))")
-    println(io, "Type    $(ispooled)$(string(eltype(dv)))")
-    println(io, "NAs     $(sum(isna(dv)))")
-    println(io, "NA%     $(round(sum(isna(dv))*100/length(dv), 2))%")
-    println(io, "Unique  $(length(unique(dv)))")
-    return
+# drop when DataFrames is julia v0.6 only (can support DataArrays >= v0.5)
+if Pkg.installed("DataArrays") < v"0.5"
```

Comment:
`method_exists` is returning `true`, I think because the function is going by `AbstractDataVector <: AbstractVector`.

I could do a manual comparison against the signatures?
```julia
julia> [m.sig for m in methods(StatsBase.describe)]
5-element Array{DataType,1}:
 Tuple{StatsBase.#describe,AbstractArray}
 Tuple{StatsBase.#describe,IO,AbstractArray{T<:Real,N}}
 Tuple{StatsBase.#describe,IO,AbstractArray}
 Tuple{StatsBase.#describe,DataFrames.AbstractDataFrame}
 Tuple{StatsBase.#describe,Any,DataFrames.AbstractDataFrame}
```

on Julia 0.5 w/ DataArrays 0.3.12
```julia
julia> [string(m.sig) == "Tuple{StatsBase.#describe,IO,DataArrays.AbstractDataArray{T,1} where T}" for m in methods(StatsBase.describe)]
5-element Array{Bool,1}:
 false
 false
 false
 false
 false
```

Julia 0.6 w/ DataArrays 0.5.1
```julia
julia> [string(m.sig) == "Tuple{StatsBase.#describe,IO,DataArrays.AbstractDataArray{T,1} where T}" for m in methods(StatsBase.describe)]
8-element Array{Bool,1}:
 false
 false
 false
  true
 false
 false
 false
 false
```

and then we could use
```julia
!any([string(m.sig) == "Tuple{StatsBase.#describe,IO,DataArrays.AbstractDataArray{T,1} where T}" for m in methods(StatsBase.describe)])
```

in place of
```julia
!method_exists(StatsBase.describe, Tuple{AbstractDataVector})
```